File: android/SellingCartLibrary/src/main/java/net/natura/sellingcartlibrary/SellingCartLibraryApplication.java
package net.natura.sellingcartlibrary;

import android.app.Application;
import android.content.Context;
import androidx.multidex.MultiDex;


public class SellingCartLibraryApplication extends Application {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: android/SellingCartLibrary/src/main/java/net/natura/sellingcartlibrary/SellingCartLibraryApplication.kt
package net.natura.sellingcartlibrary

import android.app.Application
import android.content.Context
import androidx.multidex.MultiDex

class SellingCartLibraryApplication : Application() {

    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}
